pkg/controller/kabaneroplatform/targetnamespaces.go

```diff
@@ -0,0 +1,130 @@
+package kabaneroplatform
+import (
+	"context"
+
+	kabanerov1alpha2 "github.com/kabanero-io/kabanero-operator/pkg/apis/kabanero/v1alpha2"
+	
+	"github.com/go-logr/logr"
+	
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+func reconcileTargetNamespaces(ctx context.Context, k *kabanerov1alpha2.Kabanero, cl client.Client, reqLogger logr.Logger) error {
+
+	// Rolebinding Template
+	ownerIsController := true
+	rolebindingResource := rbacv1.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kabanero-pipeline-deploy-rolebinding",
+			OwnerReferences: []metav1.OwnerReference{
+				metav1.OwnerReference{
+					APIVersion: k.TypeMeta.APIVersion,
+					Kind:       k.TypeMeta.Kind,
+					Name:       k.ObjectMeta.Name,
+					UID:        k.ObjectMeta.UID,
+					Controller: &ownerIsController,
+				},
+			},
+		},
+		Subjects: []rbacv1.Subject{
+			rbacv1.Subject{
+				Kind: "User",

```

Comment:
Should this be `Kind: "ServiceAccount"`?  I guess I don't really care, if it works this way it's fine.  I'm looking at the RBAC doc and it seems like there is a `ServiceAccount` kind.
https://kubernetes.io/docs/reference/access-authn-authz/rbac/

Reply:
yes, will fix